# # 全局参数示例
# robot_name: "dual_robot"
# max_speed: 1.0

# # 私有命名空间参数示例（通常更常用）
# fleximind_robot_node:
#   boxid_left: 0
#   boxid_right: 1
#   ip_left: "192.168.1.20"
#   ip_right: "192.168.1.30"
#   port: 10003
#   arm_type: 0
#   monitor_port: 10006
#   stop_status: False
#   left_joint_names:
#   [
#     'left_joint1', 'left_joint2', 'left_joint3', 
#     'left_joint4', 'left_joint5', 'left_joint6', 'gripper_left'
#   ]
#   right_joint_names:
#   [
#     'right_joint1', 'right_joint2', 'right_joint3', 
#     'right_joint4', 'right_joint5', 'right_joint6', 'gripper_right'
#   ]
#   timer_period: 0.1
  # 全局参数示例
robot_name: "dual_robot"
max_speed: 1.0

# 私有命名空间参数示例（通常更常用）
fleximind_robot_node:
  boxid_left: 0
  boxid_right: 1
  ip_left: "192.168.1.30"
  ip_right: "192.168.1.20"
  port: 10003
  arm_type: 0
  monitor_port: 10006
  stop_status: False
  left_joint_names:
    - left_joint1
    - left_joint2
    - left_joint3
    - left_joint4
    - left_joint5
    - left_joint6
    - gripper_left
  right_joint_names:
    - right_joint1
    - right_joint2
    - right_joint3
    - right_joint4
    - right_joint5
    - right_joint6
    - gripper_right
  timer_period: 0.1
  servo_control:
    interp_ratios: 3
    max_velocity: 1.0
    max_accel: 0.5
    max_jerk: 2.0
    serve_time: 0.004
    dp: 8000
    lookahead_time: 0.2
    buffer_size: 1000